### ISSUE TITLE
False Positive `SpacesAroundBladeRenderContent`

### ISSUE THREAD

**Issue by loganhenson** [tightenco/tlint#39]
```blade
{{-- This comment will not be present in the rendered HTML --}}
```
